sdcm/db_stats.py

```diff
@@ -413,6 +430,42 @@ def calculate_stats_average(self):
         if total_stats:
             self._stats['results']['stats_total'] = total_stats
 
+    def _cut_comments(self, result):
```

Comment:
better to call it `remove_comments` make it staticmethod/put in `utils.py`

Reply:
make at as function in utils.py.